inc/container.class.php

```diff
@@ -1733,4 +1735,29 @@ static function getSystemName($itemtype = "", $raw_name = "") {
    static function getIcon() {
       return "fas fa-tasks";
    }
+
+   static function getNameField() {
+      return 'label';
+   }
+
+   function prepareInputForClone($input) {
+      $input['itemtypes'] = json_decode(stripslashes($input['itemtypes']));
+      return $input;
+   }
+
+   public function getCloneRelations(): array
+   {
+      return [];
+   }
+
+   public function post_clone($source, $history)
+   {
+      $fields = new PluginFieldsField();
+      $fields_list = $fields->find(['plugin_fields_containers_id' => $source->getID()]);
+      foreach($fields_list as $id => $field) {
+         $fields->getFromDB($id);
+         $field['plugin_fields_containers_id'] = $this->getID();
+         $fields->clone($field, $history);
+      }
+   }
```

Comment:
I will try to see if it is possible to extend `CommonDBChild` in `PluginFieldsField`. It should prevent requiring such code.